src/Sulu/Bundle/AdminBundle/Resources/translations/admin.en.json

```diff
@@ -61,6 +61,8 @@
     "sulu_admin.list_search_placeholder": "Search…",
     "sulu_admin.error": "Error",
     "sulu_admin.warning": "Warning",
+    "sulu_admin.info": "Info",
+    "sulu_admin.success": "Success",
```

Comment:
Those are missing in `admin.de.json`, right?

Reply:
fixed